### PR TITLE
add logo to documentation

### PR DESCRIPTION
**Pull request by behackl**
Let's put our newly chosen logo prominently in the docs!

## Acknowledgement
- [x] I have read the [Contributing Guidelines](https://github.com/ManimCommunity/manim/wiki/Documentation-guidelines-(WIP))

### CODE REVIEW

**Review by leotrs — COMMENTED** [review]
Does this actually show the logo in any of the pages? Is there some other file that already imports logo-text.html?

**Comment by behackl** [comment]
> Does this actually show the logo in any of the pages? Is there some other file that already imports logo-text.html?

The Sphinx theme we are using uses our file rather than theirs; so yes: this should actually show in *every* page of the documentation. (But wait a minute or two, then the RTD build is done. 😄)

**Comment by behackl** [comment]
... and there it is!

https://manimce--577.org.readthedocs.build/en/577/

**Comment by leotrs** [comment]
Oh woah

**Comment by leotrs** [comment]
yikes this is beautiful

**Comment by leotrs** [comment]
Here it is with margin-top: 20px and margin-bottom: 20px.
2020-10-20-144123_713x274_scrot

File: docs/source/_templates/logo-text.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<img src="{{ pathto('_static/manim-logo-sidebar.png', 1) }}" alt="ManimCommunity logo" width="100%">
+<a href="{{ homepage() }}" class="text-logo">{{ theme_project_nav_name or shorttitle }}</a>
```
Comment (naveen521kk) [inline docs/source/_templates/logo-text.html:2]: A line at end of file?

**Review by naveen521kk — CHANGES_REQUESTED** [review]

**Comment by behackl** [comment]
> Here it is with margin-top: 20px and margin-bottom: 20px.
> 2020-10-20-144123_713x274_scrot

I've made the background transparent and added some more padding to top and bottom of the logo; it starts taking a lot of space on smaller screens -- personally, I like it more as-is, without the added additional padding. :-)

**Comment by leotrs** [comment]
Ah fair, didn't think of smaller screens.

File: docs/source/_templates/logo-text.html (hunk 1 above)
Comment (behackl) [reply docs/source/_templates/logo-text.html]: okay: c9685765

**Review by leotrs — APPROVED** [review]

**Comment by eulertour** [comment]
Is it possible to use an svg so that it will scale to any resolution?

**Comment by behackl** [comment]
> Is it possible to use an svg so that it will scale to any resolution?

In principle, the side bar is hard coded not to get wider than, like, 300px or so. But I agree, using the SVG directly is a bit cleaner: 3c13ceb1

**Review by eulertour — APPROVED** [review]

**Pull request merged**